integration/confidential/lib.sh

```diff
@@ -445,3 +451,77 @@ EOF
 EOF
   fi
 }
+
+###############################################################################
+
+# remote-snapshotter
+
+configure_remote_snapshotter() {
+	case "${SNAPSHOTTER:-}" in
+	"nydus")
+		configure_nydus_snapshotter
+		;;
+	*) ;;
+
+	esac
+}
+
+check_containerd_version() {
+	containerd_version=$(containerd --version | awk '{print $3}')
+	if echo $containerd_version | grep "^v1.7"; then
+		return 1
+	else
+		return 0
+	fi
+}
```

Comment:
Comment from the previous review. I wonder if 
code like:
```bash
is_containerd_support_per_runtime_snapshotter () {
   containerd_version=$(containerd --version | awk '{print $3}')
   required_version="v1.7.0"
   printf '%s\n' ${required_version} ${containerd_version} | sort --check=quiet  -V
}
```
would be better as:
- I think that `is_containerd_support_per_runtime_snapshotter` is the real thing you are interested in in this function
- Doing the version sort and compare means that if end up with containerd 2.0 installed then it will correctly pass, which this code won't when it just checks for `v1.7` substring

If you deliberately didn't do this previously as you want to pull this out as a follow on then just let me know. Thanks

Reply:
Oh, sorry, I forgot to fix it.

Reply:
Fixed.